helm/provisioner/Chart.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: v2
 name: local-static-provisioner
 description: Helm chart for the SIG Storage Local Volume Static Provisioner.
 type: application
-version: 1.0.0
+version: 2.0.0
```

Comment:
Setting `enableWindows: false` works but we'll make the v2.0.0 bump after a few more commits, could you please undo this line?

Reply:
alright reverted back to v1